src/rest_api/classes/anatomy_term/widgets/overview.clj

```diff
@@ -0,0 +1,159 @@
+(ns rest-api.classes.anatomy_term.widgets.overview
+  (:require
+   [datomic.api :as d]
+   [rest-api.classes.generic-fields :as generic]
+   [rest-api.formatters.object :as obj :refer [pack-obj]]))
+
+(def q-anatomy-functions-involved
+  '[:find ?af
+    :in $ ?at
+    :where [?ih :anatomy-function.involved/anatomy-term ?at]
+           [?af :anatomy-function/involved ?ih]])
+
+(def q-anatomy-function-not-involved
+  '[:find ?af
+    :in $ ?at
+    :where [?nih :anatomy-function.not-involved/anatomy-term ?at]
+           [?af :anatomy-function/not-involved ?nih]])
+
+(def q-anatomy-term-to-expression-cluster
+  '[:find ?e
+    :in $ ?a
+    :where [?eh :expression-cluster.anatomy-term/anatomy-term ?a]
+           [?e :expression-cluster/anatomy-term ?eh]])
+
+(defn transgenes [anatomy-term]
+  {:data (if-let [expression-patterns (:anatomy-term/expr-descendent anatomy-term)]
+           (filter
+             some?
+             (flatten
+               (for [expression-pattern expression-patterns]
+                 (for [tg (:expr-pattern/transgene expression-pattern)]
+                   (if (contains? tg :transgene/marker-for)
+                     (pack-obj tg)))))))
+   :description "transgenes annotated with this anatomy_term"})
+
+(defn expression-clusters [anatomy-term]
+  (let [db (d/entity-db anatomy-term)]
+    {:data (when-let [eids (d/q q-anatomy-term-to-expression-cluster
+                                db (:db/id anatomy-term))]
+             (for [eid eids
+                   :let [expression-cluster (d/entity db (first eid))]]
+               {:expession_cluster (pack-obj expression-cluster)
+                :description (first (:expression-cluster/description expression-cluster))}))
+     :description "expression cluster data"}))
+
+(defn term [anatomy-term]
+  {:data (if-let [holder (:anatomy-term/term anatomy-term)]
+             {:id (:anatomy-term.term/text holder)
```

Comment:
maybe pull out dup call `(:anatomy-term.term/text holder)` into a var in a let expression?